### DISCUSSION TITLE
Trying to build mina node and i get this error

### DISCUSSION THREAD

**Discussion by sbnair** [MinaProtocol/mina#10763] [Q&A]
When i am trying to build mina node from develop and also in block-producer branch i get this error :

../../../scripts/build-go-helper.sh libp2p_helper
error obtaining VCS status: exit status 128
	Use -buildvcs=false to disable VCS stamping.
make[1]: *** [Makefile:12: libp2p_helper] Error 1
make[1]: Leaving directory '/home/ubuntu/mina/src/app/libp2p_helper'
make: *** [Makefile:76: libp2p_helper] Error 2

What should i need to do,pls let me know.

**Comment by robinbb** [comment]
These repro instructions are insufficient to diagnose the issue.

**Comment by shimkiv** [comment]
Hey @sbnair try to build it using [Nix](https://github.com/MinaProtocol/mina/blob/develop/nix/README.md):
- Enter the Nix shell: `nix develop mina`
- Build executables: `MINA_COMMIT_SHA1=$(git rev-parse HEAD) DUNE_PROFILE="testnet_postake_medium_curves" dune build src/app/cli/src/mina.exe src/app/logproc/logproc.exe src/app/archive/archive.exe`

**Comment by Isaac-DeFrain** [comment]
Might be helpful
https://github.com/MinaProtocol/mina/tree/develop/src/app/libp2p_helper/README.md
